File: 1.-Two-Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n_map={}
        for i, num in enumerate(nums):
            c=target-num
            if c in n_map:
                return [n_map[c],i]
            n_map[num]=i
        return []
        
        

File: 1. Two Sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let nmap=new Map()
    
    for (let i=0;i<nums.length;i++){
        let c=target-nums[i]
        if (nmap.has(c)){
            return [nmap.get(c),i]
        }
        nmap.set(nums[i],i)
    }
    return []
};
